App/Species/KineticSpecies.lua

```diff
@@ -220,6 +220,7 @@ function KineticSpecies:fullInit(appTbl)
    self.positivity        = xsys.pickBool(tbl.applyPositivity, false)
    self.positivityDiffuse = xsys.pickBool(tbl.positivityDiffuse, self.positivity)
    self.positivityRescale = xsys.pickBool(tbl.positivityRescale, false)
+   self.nonconPositivity  = xsys.pickBool(tbl.nonconPositivity, false)
```

Comment:
Are `positivity` and `nonconPositivity` exclusive? If yes maybe there should be a check to make sure use doesn't request both?

Reply:
Will think about how to do this appropriately.